cartridge-cli.lua

```diff
@@ -1187,26 +1209,44 @@ local function form_distribution_dir(source_dir, dest_dir)
                  "normally ignored are shipped to the resulting package. ")
     end
 
-    -- deleting files matching patterns from .cartridge.ignore
-    remove_ignored(dest_dir)
+    if not opts.deprecated_build_flow then
```

Comment:
Let's move "deprecated build flow" into separate commit that will be easy to revert in future